tasks.go

```diff
@@ -174,6 +174,9 @@ func RunAllAnalysisTasks(runner Runner, path string, workers int) AnalysisResult
 				analysisResults["projects"][result.Name] = result.Results
 			} else if result.Scope == "platform" {
 				analysisResults["platform"]["platform"] = result.Results
+			} else {
+				// FIXME: we should handle this case properly, or code in a way that this conditional is gone.
```

Comment:
@philbrookes would you have a proposal for how to handle this?
We cannot simply ignore the "else" branch, to the price of implementing new checks and having them going to limbo.